portfolio/src/main/webapp/script.js

```diff
@@ -30,7 +30,16 @@ function getRandomFactAboutRoss() {
 
   // Pick a random fact.
   const fact = facts[Math.floor(Math.random() * facts.length)];
-
+  let currentFactIndex = facts.indexOf(fact);
+  if(currentFactIndex === 2 || currentFactIndex === 3) {
```

Comment:
Have comment above about why diff style needed and how does those hard code index chosen. 

Make them as global var at top as well.